teos/src/main.rs

```diff
@@ -119,6 +135,9 @@ async fn main() {
         DBM::new(path_network.join("teos_db.sql3")).unwrap(),
     ));
 
+    // Log datadir path
```

Comment:
Let's move this some lines up some it is before the options, so right after `log::info!("Config file: {:?}", &conf_file_path);`

Reply:
> Let's move this some lines up some it is before the options, so right after `log::info!("Config file: {:?}", &conf_file_path);`

If I move this some lines up it shows error as `path_network` is defined below.

Reply:
Define `path_network` first then? I don't really see the issue.

There's also the option of moving everything else down.

Reply:
> Define `path_network` first then? I don't really see the issue.
> 
> There's also the option of moving everything else down.

Done. Can I squash all the commits now as there is no issue left to solve?